Makefile

```diff
@@ -82,7 +84,5 @@ docker-push:
 	docker push ${MANAGER_IMAGE}
 
 # Build controller-gen
-# Marking this as PHONY so it will build every time (so you always have the correct version locally)
-.PHONY: $(CONTROLLER_GEN)
-$(CONTROLLER_GEN):
-	go build -o $(CONTROLLER_GEN) sigs.k8s.io/controller-tools/cmd/controller-gen
+$(CONTROLLER_GEN): $(TOOLS_DIR)/go.mod
```

Comment:
I still worry that we have potentially reintroduced drift here between the version of controller-gen we are using here and the version of controller-tools we are vendoring for the controller.

Reply:
hmm, maybe pin both with a replace via some module tidy script?

Reply:
@detiber it's not vendored anywhere else. It's only used here. I need to regen the main go.sum to make it go away.

Reply:
Updated - ptal

Reply:
I think the concern is that controller-runtime needs to match controller-tools?